modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java

```diff
@@ -1102,6 +1102,8 @@ else if (exception instanceof DuplicateLockException ||
 				"mvcPath", "/document_library/error.jsp");
 		}
 		else {
+			SessionErrors.add(actionRequest, exception.getClass());
+
```

Comment:
it's to show an generic error to the page, in a similar way as d&d with errors on back (as the SQL error if the file has an emoji on the name)

https://user-images.githubusercontent.com/47524751/172623832-4d06cdfd-e09c-4f93-aa76-15c8315f3c8e.mp4

If not present it will fail the same but no message will be shown so maybe it's a small improvement

https://user-images.githubusercontent.com/47524751/172625545-11e785db-3f1f-4b42-a702-0c473812dd2a.mp4

Reply:
@adolfopa 

> can you add this change in a different commit?

Do you mean in another PR?

Reply:
Sorry, not sure why I thought everything was in a single commit. Nevermind.